### PR TITLE
 [FEATURE] notifications from visual rules

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/mock/perseo_mock.py [hunk 1]
```diff
@@ -106,7 +106,6 @@ def do_POST (s):
                    print "post counter: " + str(post_number)
        except Exception, e:
            print "WARN - "+ str(e)
-           #print mock_config.WARN + mock_config.CONTENT_LENGTH_WARN_MSG
```
Comment (fgalan) [inline test/mock/perseo_mock.py]: Leftover?

Comment (iariasleon) [inline test/mock/perseo_mock.py]: In the current file, this line is erased, I thing, you do see the old version.

**Comment by jcalderin** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**